src/styles/pages/_connectPage.scss

```diff
@@ -0,0 +1,53 @@
+.msteams-connect {
+    &__title {
+        position: absolute;
```

Comment:
prevent `absolute` position

any position property should be used only when there is a need to modify the normal flow of the HTML page, for placing title here, I don't think we really need it, we can sync if needed